src/resources/MachineLearning/SmartSnippetsConfiguration/SmartSnippetsConfiguration.ts

```diff
@@ -1,10 +1,22 @@
 import API from '../../../APICore';
 import {New} from '../../BaseInterfaces';
 import Resource from '../../Resource';
-import {SmartSnippetsConfigurationModel} from './SmartSnippetsConfigurationInterfaces';
+import {
+    SmartSnippetsConfigurationModel,
+    SmartSnippetsContentFields,
+    SmartSnippetsContentFieldsParams,
+    SmartSnippetsDocumentGroupPreview,
+    SmartSnippetsDocumentGroupPreviewParams,
+    SmartSnippetsDocumentTypes,
+    SmartSnippetsDocumentTypesParams,
+} from './SmartSnippetsConfigurationInterfaces';
 
 export default class SmartSnippetsConfiguration extends Resource {
-    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/configuration/smartsnippets/model`;
+    static rootUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/configuration/smartsnippets`;
+    static baseUrl = `${SmartSnippetsConfiguration.rootUrl}/model`;
```

Comment:
Could also rename this instead of adding `rootUrl` but that would be breaking. Open to doing so if that's preferred, or something else.

Reply:
Having both `baseUrl` and `rootUrl` might be a bit confusing. the breaking change would be something like this:
```
static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/configuration/smartsnippets`;
static modelUrl = `${SmartSnippetsConfiguration.baseUrl}/model`;
```
without `rootUrl` altogether.